ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -245,13 +110,13 @@ public File getKeyStorePasswordFile() {
   private Path dataPath;
 
   @ArgGroup(exclusive = false)
-  private TlsServerOptions tlsServerOptions;
+  private PicoCliTlsServerOptions picoCliTlsServerOptions;
 
   @ArgGroup(exclusive = false)
-  private TlsClientCertificateOptions clientTlsCertificateOptions;
+  private PicoCliTlsClientCertificateOptions clientTlsCertificateOptions;
 
   @ArgGroup(exclusive = false)
-  private Web3ProviderTrustStore web3ProviderTrustStore;
+  private PicoCliWeb3ProviderTrustStore picoCliWeb3ProviderTrustStore;
```

Comment:
I think i'd prefer these named in their old formats, not sure PicoCli really adds anything to the class names?

Reply:
I was thinking these are _explicitly_ PicoCLI versions of the config class - i.e. populated from a PicoCli cmdline ...

Reply:
yep sure, maybe package if its important (they're in a pretty logical group), but i'm not sure it really adds much (i can't believe i'm saying that heh)
I just think it detracts from the naming because its all using picoCli, and we haven't called the baseCommand class picoCliEthSignerBaseCommand... not that we should either.... just as an example...

Reply:
RIght - so PicoCli might be able to go, but then I have to replace it with something (as they all have to differentiate from the underlying interface ... could go with Cmdline<classname>  - but _if_ we ever moved away from PicoCli, it wouldn't fit.
Any other suggestions?

Reply:
I think we can keep PicoCli